### ISSUE TITLE
Consider placement of the search window

### ISSUE THREAD

**Issue by onurmercury** [Vanilla-OS/legendary-desktop-feedback#6]
**Is your feature request related to a problem? Please describe.**
Search window should not move depending on search result. That just makes search box and search results hard to follow.

**Describe the solution you'd like**
Pre-defined height for the search window might help.

**Describe alternatives you've considered**
None.

**Additional context**
![Screenshot_20240401-171725_Discord](https://github.com/Vanilla-OS/legendary-desktop-feedback/assets/141233985/cde298d0-dc59-4ac0-8343-dae5947258db)
![Screenshot_20240401-171818_Discord](https://github.com/Vanilla-OS/legendary-desktop-feedback/assets/141233985/989960cd-6d22-4549-83ad-c7772fbce6cf)
![Screenshot_20240401-171758_Discord](https://github.com/Vanilla-OS/legendary-desktop-feedback/assets/141233985/2846f5bd-4303-4ede-898e-c2cce41fa5e6)

**Comment by SethbotStar** [comment]
Hmm, I think it gets rid of a bit of unnecessary screen clutter. In fact I think this feature should be extended to the horizontal dimension.
